VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/venue-list/venue-list.component.ts

```diff
@@ -52,7 +52,7 @@ export class VenueListComponent implements OnInit {
 
     getFiltersCourtRoomsAccounts(response: CourtRoomsAccountResponse[]) {
         if (this.venuesSelected) {
-            this.filterCourtRoomsAccounts = response.map(x => new CourtRoomsAccounts(x.venue, x.court_rooms, true));
+            this.filterCourtRoomsAccounts = response.map(x => new CourtRoomsAccounts(x.first_name, x.last_names, true));
```

Comment:
What is this change? Why would we use first_name? And why would last_names be a thing?